examples/website-demos/social-messaging/src/components/Message/SocialMessageUI.tsx

```diff
@@ -0,0 +1,120 @@
+import {
+  Avatar,
+  MessageText,
+  MessageTimestamp,
+  MessageUIComponentProps,
+  useChannelStateContext,
+  useChatContext,
+  useMessageContext,
+} from 'stream-chat-react';
+import { DeliveredCheckmark, DoubleCheckmark } from '../../assets';
+
+import {
+  SocialAttachmentType,
+  SocialChannelType,
+  SocialCommandType,
+  SocialEventType,
+  SocialMessageType,
+  SocialReactionType,
+  SocialUserType,
+} from '../ChatContainer/ChatContainer';
+
+import './SocialMessageUI.scss';
+
+export const SocialMessage: React.FC<
+  MessageUIComponentProps<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
+  >
+> = (props) => {
+  const { channel } = useChannelStateContext<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
+  >();
+  const { client } = useChatContext<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
+  >();
+  const { isMyMessage, message, readBy } = useMessageContext<
+    SocialAttachmentType,
+    SocialChannelType,
+    SocialCommandType,
+    SocialEventType,
+    SocialMessageType,
+    SocialReactionType,
+    SocialUserType
+  >();
+
+  const members = Object.values(channel.state.members).filter(
+    ({ user }) => user?.id !== client.userID,
+  ).length;
+
+  const myMessage = isMyMessage();
+
+  const readByMembers = readBy?.filter((user) => user.id !== client.user?.id);
+  const readByMembersLength = readByMembers?.length === 0 ? undefined : readByMembers?.length;
```

Comment:
do we need this ternary? can we just use `readByMembers?.length` without creating the new variable? the way it's written the value of readByMembersLength will be either undefined or a positive number. If we don't use the ternary the value will with be undefined or zero or a positive number. zero is still a falsey value so i'm not sure it will affect however this variable is used later on